WORKSPACE

```diff
@@ -19,13 +19,29 @@ swc_register_toolchains(
     swc_version = "v1.2.168",
 )
 
+load("@aspect_bazel_lib//lib:repositories.bzl", "DEFAULT_YQ_VERSION", "register_yq_toolchains")
+
+register_yq_toolchains(
+    version = DEFAULT_YQ_VERSION,
+)
+
 load("@rules_nodejs//nodejs:repositories.bzl", "nodejs_register_toolchains")
 
 nodejs_register_toolchains(
     name = "node16",
     node_version = "16.9.0",
 )
 
+load("@aspect_rules_js//js:npm_import.bzl", "translate_pnpm_lock")
+
+# This is used to generate code in this repo, but otherwise unused
+# at runtime or by users directly.
+translate_pnpm_lock(
+    name = "swc_cli",
+    node_repository = "node16",
+    pnpm_lock = "@aspect_rules_swc//swc:pnpm-lock.json",
```

Comment:
Should this be a yaml file?

Reply:
yeah, I had vendored it in before the yq support landed